tasks/bump.js

```diff
@@ -77,14 +82,12 @@ module.exports = function(grunt) {
 
     var globalVersion; // when bumping multiple files
     var gitVersion;    // when bumping using `git describe`
-    var VERSION_REGEXP = /([\'|\"]?version[\'|\"]?[ ]*:[ ]*[\'|\"]?)([\d||A-a|.|-]*)([\'|\"]?)/i;
-
+    var VERSION_REGEXP = new RegExp('([\\\'|\\\"]?version[\\\'|\\\"]?[ ]*:[ ]*[\\\'|\\\"]?)(\\\d+\\\.\\\d+\\\.\\\d+(-'+opts.prereleaseName+'\\\.\\\d+)?(-\\\d+)?)[\\\d||A-a|.|-]*([\\\'|\\\"]?)', 'i');
```

Comment:
If you get rid of all the escapes (that are there so we can add a variable) it's much loess 'scary' looking. I tried to write a couple of tests, but am unfamiliar with testing node modules.
I could just copy the regex over to a test for now (though not a great strategy). If you have any suggestions on how to add tests for the regex I'd be happy to flush them out.  
I had tested this quite a bit manually.  Let me know if there's anything I can do to help out.